var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -55,3 +55,6 @@ def url_for_version(self, version):
         else:
             url = "https://github.com/AIDASoft/podio/archive/v%s-%s-%s.tar.gz" % (major, minor, patch)
         return url
+
+    def setup_run_environment(self, spack_env):
+        spack_env.prepend_path('PYTHONPATH', self.prefix.python)
```

Comment:
```suggestion
    def setup_run_environment(self, env):
        env.prepend_path('PYTHONPATH', self.prefix.python)
```
For consistency with other packages.